syntax = "proto3";

package pb;

import "datadog_lambda/pb/span.proto";

message APITrace {
	uint64 traceID = 1;
	repeated Span spans = 2;
	int64 startTime = 6;
	int64 endTime = 7;
}
